lib/src/idirectus_api_manager.dart

```diff
@@ -4,7 +4,7 @@ import 'package:http/http.dart';
 
 abstract class IDirectusApiManager {
   Future<DirectusLoginResult> loginDirectusUser(
-      String username, String password);
+      String username, String password, String? oneTimePassword);
```

Comment:
This is not the same signature as the implementation where you set the parameter as optional and here it is mandatory. Using named parameter should help as it won't be building with differences like this.

Reply:
Changed from unnamed optional [String? oneTimePassword] to named optional {String? oneTimePassword}.